sample-authz/src/main/java/net/trajano/ms/example/authz/JwtGrantHandler.java

```diff
@@ -17,6 +17,7 @@
 import javax.ws.rs.core.UriBuilder;
 import javax.ws.rs.core.UriBuilderException;
 
+import com.hazelcast.core.HazelcastInstance;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'com.hazelcast.core.HazelcastInstance'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)